tests/chainerx_tests/unit_tests/routines_tests/test_creation.py

```diff
@@ -339,54 +339,15 @@ def test_asarray_with_device(device):
     array_utils.check_device(a, device)
 
 
-@chainerx.testing.numpy_chainerx_array_equal()
-@pytest.mark.parametrize('padding', [False, True])
-def test_ascontiguousarray_from_numpy_array(xp, shape, dtype, padding):
-    obj = array_utils.create_dummy_ndarray(
-        numpy, shape, dtype, padding=padding)
-    a = xp.ascontiguousarray(obj)
-    if xp is chainerx:
-        assert a.is_contiguous
-    return a
-
-
-@chainerx.testing.numpy_chainerx_array_equal()
-@pytest.mark.parametrize_device(['native:0', 'cuda:0'])
-@pytest.mark.parametrize('obj', _array_params(_array_params_list))
-@chainerx.testing.parametrize_dtype_specifier(
-    'dtype_spec', additional_args=(None, Unspecified))
-def test_ascontiguousarray_from_tuple_or_list(xp, device, obj, dtype_spec):
-    if xp is numpy and isinstance(dtype_spec, chainerx.dtype):
-        dtype_spec = dtype_spec.name
-    # Skip nan/inf -> integer conversion that would cause a cast error.
-    if (not _is_all_finite(obj)
-            and dtype_spec not in (None, Unspecified)
-            and chainerx.dtype(dtype_spec).kind not in ('f', 'c')):
-        return chainerx.testing.ignore()
+def test_ascontiguousarray_from_chainerx_array_device():
+    with chainerx.using_device(chainerx.get_device('native:0')):
+        dev = chainerx.get_device('native:1')  # Non default one
+        assert chainerx.get_default_device() is not dev
 
-    if dtype_spec is Unspecified:
-        a = xp.ascontiguousarray(obj)
-    else:
-        a = xp.ascontiguousarray(obj, dtype=dtype_spec)
-
-    if xp is chainerx:
-        assert a.is_contiguous
-    return a
-
-
-@pytest.mark.parametrize_device(['native:0', 'cuda:0'])
-@pytest.mark.parametrize('padding', [False, True])
-def test_ascontiguousarray_from_chainerx_array(device, shape, dtype, padding):
```

Comment:
Shoudn't this test be kept?

Reply:
Right! What I needed was to fix the numpy array -> chainerx array.